docs/user/faq.rst

```diff
@@ -436,22 +436,23 @@ around:
     def on_post(self, req, resp):
         pass
 
+.. _faq_override_404_500_handlers:
 
 How can I write a custom handler for 404 and 500 pages in falcon?
 ------------------------------------------------------------------
 When a route can not be found for an incoming request, Falcon uses a default
-responder that simply raises an instance of :attr:`falcon.HTTPNotFound`. You
-can use :meth:`falcon.App.add_error_handler` to register a custom error handler
-for this exception type. Alternatively, you may be able to configure your web
-server to transform the response for you (e.g., using Nginx's ``error_page``
-directive).
+responder that simply raises an instance of :class:`~.HTTPRouteNotFound`, which
+the framework will in turn render as a 404 response. You can use
+:meth:`falcon.App.add_error_handler` to override the default handler for this
+exception type (or for its parent type,  :class:`~.HTTPNotFound`).
+Alternatively, you may be able to  configure your web server to transform the
```

Comment:
Not that it matter much in reStructured Text, but there in an extraneous space after `able to`.

Reply:
fixed